Packages/Web/lib/util.flp

```diff
@@ -459,3 +459,46 @@ end
 func trackName(propertyName, id)
   return "\(id)_\(propertyName.snakeCased())Track"
 end
+
+func trackOffsets(_ times, _ duration)
+    var offsets = []
+    if times[0] != 0
+        offsets = [0]
+    end
+    for time in times
+        offsets = offsets.appending(time / duration)
+    end
+    if offsets[offsets.count - 1] != 1
+        offsets = offsets.appending(1)
+    end
+    return offsets
+end
+
+func trackValues(property, _ keyvalues, layer, imageFolderPath, importImage, _ duration)
+    var values = []
+    for keyvalue in keyvalues
+        var value = cssValue(property: property, value: keyvalue.value, layer: layer, imageFolderPath: imageFolderPath, importImage: importImage)
+        values = values.appending(value)
+    end
+
+    values = addTrailingKeyvalue(keyvalues, duration, values, values[values.count - 1])
+    values = addInitialValue(keyvalues, duration, values, values[0])
+
+    return values
+end
+
+func addTrailingKeyvalue(_ keyvalues, _ duration, _ arr, _ appendingValue)
+     if keyvalues[keyvalues.count - 1].time * 1000 < duration
+        arr = arr.appending(appendingValue)
+        return arr
+    end
+    return arr 
+end
```

Comment:
I think we might not need this anymore now that we're using the simulated keyvalues.